integrations/datadog/.port/resources/blueprints.json

```diff
@@ -0,0 +1,191 @@
+[
+  {
+    "identifier": "datadogMonitor",
+    "description": "This blueprint represents a datadog monitor",
+    "title": "Datadog Monitor",
+    "icon": "Datadog",
+    "schema": {
+      "properties": {
+        "monitorType": {
+          "type": "string",
+          "title": "Monitor Type"
+        },
+        "tags": {
+          "type": "array",
+          "title": "Tags"
+        },
+        "overallState": {
+          "type": "string",
+          "title": "Overall state",
+          "enum": ["Alert", "Ignored", "No Data", "OK", "Skipped", "Unknown", "Warn"],
+          "enumColors": {
+            "Alert": "red",
+            "Ignored": "darkGray",
+            "No Data": "lightGray",
+            "OK": "green",
+            "Skipped": "yellow",
+            "Unknown": "purple",
+            "Warn": "orange"
+          }
+        },
+        "priority": {
+          "type": "string",
+          "title": "Priority"
+        },
+        "thresholds": {
+          "type": "object",
+          "title": "Thresholds"
+        },
+        "createdBy": {
+          "type": "string",
+          "title": "Creator"
+        },
+        "createdAt": {
+          "title": "Created At",
+          "type": "string",
+          "format": "date-time"
+        },
+        "updatedAt": {
+          "title": "Updated At",
+          "type": "string",
+          "format": "date-time"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {}
+  },
+  {
+    "identifier": "datadogService",
+    "description": "This blueprint represents a Datadog service",
+    "title": "Datadog Service",
+    "icon": "Datadog",
+    "schema": {
+      "properties": {
+        "contacts": {
+          "type": "array",
+          "title": "Contacts"
```

Comment:
add property application

Reply:
Hey @Tankilevitch, please elaborate on this